.storybook/theme-selector/index.js

```diff
@@ -53,8 +53,13 @@ export const withThemeSelector = makeDecorator({
     channel.emit(PARAMS_EVENT, parameters);
     return (
       <ThemeProvider theme={theme}>
-        {!parameters.disable && <CarbonGlobalTokensProvider />}
-        {getStory(context)}
+        {parameters.disable ? (
+          getStory(context)
+        ) : (
+          <CarbonScopedTokensProvider>
+            {getStory(context)}
+          </CarbonScopedTokensProvider>
+        )}
```

Comment:
When passing `disabled: true` the decorator will not run. We do not need to handle this.

![image](https://user-images.githubusercontent.com/2328042/140777448-eb373695-3217-4354-8ac5-411af4e17641.png)
```suggestion
           <CarbonScopedTokensProvider>
            {getStory(context)}
          </CarbonScopedTokensProvider>
```